ring-jetty-adapter/src/ring/adapter/jetty.clj

```diff
@@ -189,6 +190,10 @@
   :key-password           - the password to the keystore
   :keystore-scan-interval - if not nil, the interval in seconds to scan for an
                             updated keystore
+  :thread-pool            - custom thread pool instance for Jetty to use
+                            takes precedence over :max-threads, :min-threads,
+                            :max-queued-requests, :thread-idle-timeout and
+                            :daemon?
```

Comment:
Can you remove these lines? I think this can be explained elsewhere.

Reply:
Removed.